services/fhir.ts

```diff
@@ -276,3 +380,49 @@ export function getConcepts(valueSetId: string, params?: SearchParams): Promise<
         params: { ...params },
     });
 }
+
+export async function applyFHIRService<T, F>(request: AxiosRequestConfig): Promise<RemoteDataResult<T, F>> {
+    return service(request);
+}
+
+const toCamelCase = (str: string): string => {
+    const withFirstLowerLetter = str.charAt(0).toLowerCase() + str.slice(1);
+    return withFirstLowerLetter.replace('-', '');
+};
+
+export function transformToBundleEntry<R extends AidboxResource>(config: AxiosRequestConfig): BundleEntry<R> | null {
+    const { method, url, data, params, headers = [] } = config;
+
+    if (!method || !url) {
+        return null;
+    }
+    const request = {
+        method,
+        url: typeof params === 'object' && params !== null ? url + '?' + buildQueryParams(params) : url,
+    };
+
+    ['If-Modified-Since', 'If-Match', 'If-None-Match', 'If-None-Exist'].forEach((header) => {
```

Comment:
for If-None-Match you should make buildQueryParams

e.g. IfNoneMatch: identifier=1&name=test